tests/integration/galley/analyze_test.go

```diff
@@ -220,7 +220,7 @@ func expectMessages(t *testing.T, g *GomegaWithT, outputLines []string, expected
 func expectNoMessages(t *testing.T, g *GomegaWithT, output []string) {
 	t.Helper()
 	g.Expect(output).To(HaveLen(1))
-	g.Expect(output[0]).To(ContainSubstring(cmd.NoIssuesString))
+	g.Expect(output[0]).To(ContainSubstring("No validation issues found when analyzing"))
```

Comment:
Nit but I'd rather this be a constant in cmd just to prevent someone needing to run the tests, realize there's a string, search for it, change it, etc.

Reply:
I agree that that is annoying, but it's somewhat mitigated by the fact that the tests are run automatically in CI (so you don't _have_ to run it). Also using a constant is potentially dangerous here because it makes it really easy to introduce a typo and the test will always pass.

I could potentially make the test stronger by checking for the namespace in question as well since right now we're doing a fairly weak string contains match.

Reply:
Up to you. I think it's fine to merge as-is.